Jenkinsfile

```diff
@@ -2,8 +2,10 @@ import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 
 full_ci = env.BRANCH_NAME == 'master' || pullRequest.labels.contains('FullCI')
 
-// Due to their long runtime, we skip several tests in sanitizer builds.
-tests_excluded_in_sanitizer_builds = 'SQLiteTestRunnerEncodings/*:TPCDSTableGeneratorTest.GenerateAndStoreRowCounts:TPCHTableGeneratorTest.RowCountsMediumScaleFactor'
+// Due to their long runtime, we skip several tests in sanitizer and MacOS builds.
+tests_excluded_in_sanitizer_builds = 'SQLiteTestRunnerEncodings/*:TPCDSTableGeneratorTest.GenerateAndStoreRowCounts:TPCHTableGeneratorTest.RowCountsMediumScaleFactor:SSBTableGeneratorTest.GenerateAndStoreRowCounts'
+
+tests_excluded_in_mac_builds = '--gtest_filter=\"-TPCCTest*:TPCDSTableGeneratorTest.*:TPCHTableGeneratorTest.RowCountsMediumScaleFactor:SSBTableGeneratorTest.GenerateAndStoreRowCounts:.CompareToSQLite/Line1*WithLZ4\"'
```

Comment:
Why the LZ4 exclusion on macs? Too slow? Then buy a new macbook.

And can we use the same format (with or without `--gtest_filter`)?

Reply:
> Why the LZ4 exclusion on macs? Too slow? Then buy a new macbook.

Well, since this in the `JenkinsFile`, you must admit that my Mac is not the problem. If you carefully looked, you would have seen that these were already excluded in the Mac CI stages and I only copied that. If you feel we should allow LZ4 in the Mac stages, I have no problem to remove this exception.

> And can we use the same format (with or without `--gtest_filter`)?
Guess we can

Reply:
> If you carefully looked

That's a rhetorical question, isn't it?

> you must admit that my Mac is not the problem

That might be true, but that's not a reason to not buy a new mac.

I would re-add the LZ4 tests since it's a relevant test. I should be the main bottleneck of the CI, even for the slower Intel.